### ISSUE TITLE
ImportError:undefined symbol:

### ISSUE THREAD

**Issue by Zhizh1** [iamalbert/pytorch-wordemb#6]
I downloaded and installed torchwordemb-0.0.9 via python setup.py install.and the i got an error related to return {vocab, dest};,I change the original code to return VocabAndTensor(vocab, dest);, and run python setup.py install again.then i install torchwordemb-0.0.9 successful；
but when i import torchwordemb，i meet an error ：
```
 import torchwordemb
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: /usr/local/lib/python3.6/dist-packages/torchwordemb-0.0.9-py3.6-linux-x86_64.egg/torchwordemb.cpython-36m-x86_64-linux-gnu.so: undefined symbol: _ZN6caffe26detail37_typeMetaDataInstance_preallocated_29E
```

i have found some solutions but it didn't work for me, anyone meet this problem?

**Comment by ShuangqiZhu** [comment]
Do you import torch first?
import torch
import torchwordemb

**Comment by Zhizh1** [comment]
> Do you import torch first?
> import torch
> import torchwordemb

yes,I have import torch before import torchwordemb.This question seems to be related to the version of torch.When I switched to a computer with a different torch version, there was no problem.

**Comment by petartonchev** [comment]
Downgrading to torch==1.4.0 solved the issue for me.
